x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/notification/NotificationService.java

```diff
@@ -30,7 +30,7 @@ public NotificationService(Settings settings, String type) {
         this.type = type;
     }
 
-    protected synchronized void setAccountSetting(Settings settings) {
+    public synchronized void loadSettings(Settings settings) {
```

Comment:
how about naming this `reload()` as well just like in `ReloadablePlugin` plugin?

Reply:
The only reason i did not name it reload is because it is used to init settings as well, but ill change it to `reload`.

Reply:
153a97b